app.rb

```diff
@@ -0,0 +1,117 @@
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './rental'
+require './book'
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def choose_option(input)
+    case input
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_rentals
+    else
+      invalid_option
+    end
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    puts
+  end
+
+  def list_people
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id} Age: #{person.age}" }
+    puts
+  end
+
+  def create_person
+    print 'Do you want to create a student(1) or a teacher(2)? [Input the number]:'
+    input = gets.chomp.to_i
+    case input
+    when 1
+      create_student
+    when 2
+      create_teacher
+    end
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.upcase == 'Y'
+    gets
+    @people << Student.new(name, age, parent_permission)
```

Comment:
- [ ] This line should've been placed in the 'create_person()' method. Check best practices and SOLID lesson [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid).